src/app.js

```diff
@@ -557,6 +572,18 @@ var run = function () {
         files.addReadOnly(url, content)
         cb(null, content)
       })
+    } else if ((match = /^(ipfs:\/\/?.+)/.exec(url))) {
+      $('#output').append($('<div/>').append($('<pre/>').text('Loading ' + url + ' ...')))
+      handleIPFS(match[1], function (err, content) {
+        if (err) {
+          cb('Unable to import "' + url + '": ' + err)
```

Comment:
A planned to address that in a future PR. The only thing I've did in the direction is to move the log 😉